docs/rst/notes/notes.rst

```diff
@@ -1,4 +1,33 @@
 .. _release_notes:
 
-Release notes
+Version 0.1.0
 =============
+
+This first release includes the following features:
+
+Fast DDS QoS Profiles Manager Library
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1. Library infrastructure.
+2. Implementation of several `set` functions:
+    - DomainParticipant name.
+    - DomainParticipant external locators (default and metatraffic): externality, address, and port.
+    - DomainParticipant initial peers: address and port.
+    - DomainParticipant lease duration and lease announcements.
+    - DomainParticipant use of custom transports disabling builtin transports.
+    - Reliability and Durability QoS for DataReader and DataWriter profiles.
+    - Default profile attribute for DomainParticipant, DataReader and DataWriter.
+
+Fast DDS QoS Profiles Manager CLI
```

Comment:
Include Transport descriptor `set` methods:
```suggestion
    - DomainParticipant use of custom transports disabling builtin transports.
    - Reliability and Durability QoS for DataReader and DataWriter profiles.
    - Default profile attribute for DomainParticipant, DataReader and DataWriter.
    - Transport descriptor kind.
    - Transport descriptor interface whitelist.

Fast DDS QoS Profiles Manager CLI
```

Reply:
Yep. I will modify the release notes in Github too. Thanks!